test/jdk/javax/net/ssl/TLSCommon/interop/JdkServer.java

```diff
@@ -142,7 +142,7 @@ public boolean isAlive() {
 
     @Override
     public void close() throws IOException {
-        if (isAlive()) {
+        if (!serverSocket.isClosed()) {
```

Comment:
isAlive can be overridden with a more complex implementation, whereas the only default requirement to close the socket in this case is checking it is not already closed.